app/repositories/ApprovalRepository.scala

```diff
@@ -87,8 +88,16 @@ class ApprovalRepositoryImpl @Inject() (implicit mongoComponent: ReactiveMongoCo
     //$COVERAGE-ON$
   }
 
-  def approvalSummaryList(): Future[RequestOutcome[List[ApprovalProcessSummary]]] = {
-    val selector = Json.obj("meta" -> Json.obj("$exists" -> true))
+  def approvalSummaryList(criteria: SummaryListCriteria): Future[RequestOutcome[List[ApprovalProcessSummary]]] = {
+
+    val twoEyeRestriction = Json.obj("meta.reviewType" -> Constants.ReviewType2i)
```

Comment:
As these are constants, you could add them as predefined on the ApprovalRepository trait